### PR TITLE
Email timeout

### PR DESCRIPTION
**Pull request by InnaDrapii**

### CODE REVIEW

File: src/main/java/pickleib/utilities/email/EmailAcquisition.java [hunk 1]
```diff
@@ -20,13 +21,37 @@ public EmailAcquisition(EmailInbox emailInbox){
         this.emailInbox = emailInbox;
     }
 
+    static long emailAcquiringTimeout = Long.parseLong(ContextStore.get("email-acquiring-timeout", "45000"));
```
Comment (Umutayb) [inline src/main/java/pickleib/utilities/email/EmailAcquisition.java:24]: email-acquisition-timeout could make a better key

File: src/main/java/pickleib/utilities/email/EmailInbox.java [hunk 2]
```diff
@@ -28,6 +29,7 @@ public class EmailInbox {
     String secureCon;
     String email;
     String emailApplicationPassword;
+    static long emailAcquiringTimeout = Long.parseLong(ContextStore.get("email-acquiring-timeout", "45000"));
```
Comment (Umutayb) [inline src/main/java/pickleib/utilities/email/EmailInbox.java:32]: Why have the same variable declared twice?

**Review by Umutayb — CHANGES_REQUESTED** [review]

File: src/main/java/pickleib/utilities/email/EmailAcquisition.java (hunk 1 above)
Comment (InnaDrapii) [reply src/main/java/pickleib/utilities/email/EmailAcquisition.java]: fixed

File: src/main/java/pickleib/utilities/email/EmailInbox.java (hunk 2 above)
Comment (InnaDrapii) [reply src/main/java/pickleib/utilities/email/EmailInbox.java]: The first variable is declared in EmailAcquisition.class, the second one is in EmailInbox.class.

**Comment by Umutayb** [comment]
@InnaDrapii thank you for the improvements!

**Review by Umutayb — APPROVED** [review]

**Pull request merged**